lms/services/http.py

```diff
@@ -85,7 +87,7 @@ def request(self, method, url, timeout=(10, 10), **kwargs):
             )
             response.raise_for_status()
         except RequestException as err:
-            raise HTTPError(response) from err
+            raise ExternalRequestError(response=response) from err
```

Comment:
`ExternalRequestError` accepts multiple kwargs of which `response` is not the first so `(response=response)` is needed rather than just `(response)`